VERSION

```diff
@@ -0,0 +1,6 @@
+VERSION_MAJOR=2
+VERSION_MINOR=0
+VERSION_PATCH=0
```

Comment:
I would probably merge these into one value, maybe just VERSION or VERSION_SHORT as it would be much easier to sed or replace later on in a script. Plus I think most people read it as one set of values, not 3 individual values.

Reply:
done